File: JS-Basics/Conditional Statements/Lab/07.js
function calculateArea(shape,...numbers) {
    if (shape === "square") {
        let side = numbers[0]
        console.log((side * side).toFixed(3));
    } 
    else if (shape === "rectangle") {
        let length =numbers[0]
        let width = numbers[1]
        console.log((length * width).toFixed(3));
    } 
    else if (shape === "circle") {
        let radius = numbers[0]
        console.log((Math.PI * radius * radius).toFixed(3));
    } 
    else if (shape === "triangle") {
        let base = numbers[0]
        let height = numbers[1]
        console.log(((base * height) / 2).toFixed(3));
    } 
    
}

File: Basics/Conditional Statements/Lab/07.py
from math import pi
shape = input()

if shape == "square":
    length = float(input())
    area = length * length
    print(f"{area:.3f}")
elif shape == "rectangle":
    length = float(input())
    width = float(input())
    area = length * width
    print(f"{area:.3f}")
elif shape == "circle":
    radius = float(input())
    area = radius * radius * pi
    print(f"{area:.3f}")
elif shape == "triangle":
    length = float(input())
    height = float(input())
    area = length * height / 2
    print(f"{area:.3f}")
